.travis.yml

```diff
@@ -228,7 +228,7 @@ matrix:
 
     - python: 2.7
       name: 'Frontend [build]'
-      env: TEST_SUITE=js-build
+      env: TEST_SUITE=js-build NODE_ENV=production
```

Comment:
This will use up a bit more CPU (i.e. for minification), but seems preferable to run production-like builds anyway.